waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -401,11 +427,15 @@ method getMessages*(s: PostgresDriver,
                     cursor = none(ArchiveCursor),
                     startTime = none(Timestamp),
                     endTime = none(Timestamp),
+                    hashes: seq[WakuMessageHash] = @[],
                     maxPageSize = DefaultPageSize,
                     ascendingOrder = true):
                     Future[ArchiveDriverResult[seq[ArchiveRow]]] {.async.} =
-
-  if contentTopicSeq.len == 1 and
+  let hexHashes = hashes.mapIt(toHex(it))
+  
+  # TODO find why prepared statement returns no msg
+  #[ if contentTopicSeq.len == 1 and
```

Comment:
Interesting. We need to make sure the prepared statements queries still work. DM me if need me to review something :P

Reply:
ops forgot about this one :P